features/2025-07-radius-resource-types-contribution.md

```diff
@@ -0,0 +1,292 @@
+# Contributing Radius Resource Types 
+
+* **Author**: [Reshma Abdul Rahim](https://github.com/reshrahim)
+
+## Topic Summary
+
+This feature specification defines the experience for community members to contribute new resource types to the Radius ecosystem. Building on the foundation of Radius resource types, this feature specification provides detailed guidance for contributing to all categories of Radius resource types including core types, database resources, cache resources, messaging resources, Dapr resources, etc. The specification establishes clear pathways for community members to contribute resource types and Recipes, enabling a vibrant ecosystem of shared resources that accelerates Radius adoption.
+
+### Top level goals
+
+- **Democratize resource type contribution**: Enable community members to easily contribute new resource types without requiring deep knowledge of Radius internals or Go programming
+- **Establish clear contribution pathways**: Define distinct workflows for different types of resource contributions 
+- **Enable ecosystem growth**: Foster a vibrant community marketplace of resource types that accelerates Radius adoption
+
+### Non-goals (out of scope)
+
+- Implementation of automated testing frameworks for resource types and/or CI/CD pipelines.
```

Comment:
I think this should be in scope, as this is the key value of contributing and consuming those contributions.